Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:latest
+
+ARG VERSION
+
+WORKDIR /usr/local/github-changelog
+
+RUN apt-get update
+RUN apt-get install --assume-yes ca-certificates curl
```

Comment:
Why do we need these dependencies?

Reply:
They make easier to install certificates later on. They are not necessarily required for this image though

Reply:
If they are not required then we should remove it then. But as @raszi's comment states maybe we should download `boot`, invoke it then remove it from the image: https://github.com/adzerk-oss/boot-clj-docker-image/blob/master/Dockerfile
We can keep the version ARG loading in the release script though.

Reply:
Hm, ok I will rework it then.

Reply:
Since Docker has a layer based filesystem using the `boot-clj-docker-image` and then removing the unnecessary items won't reduce the size of the image.

Instead of that we should install the required dependencies, build the jar, and remove the build dependencies in one step.

Reply:
Oh yeah, I just meant that as an example of how to install boot in Docker.

Reply:
Actually this can be done with mutlistage dockerfile which (as I could see) solves exactly this problem.